boards/PDM/Inc/App/App_Efuse.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "App_SharedExitCode.h"
+#include <stdbool.h>
```

Comment:
Standard library should be declared on top 
```suggestion
#include <stdbool.h>
#include "App_SharedExitCode.h"
```